rust/libnewsboat/src/configpaths.rs

```diff
@@ -82,24 +111,24 @@ impl ConfigPaths {
 
         // hitting this branch means we found a home directory
         // we can now safely call unwrap on all other functions in the dirs crate
-
         config_paths.env_home = env_home.unwrap();
+
+        // This can't panic because we've found the home dir.
+        // This should be replaced with proper error handling after this is not used by c++ anymore
```

Comment:
This is just a moved code comment, so feel free to ignore: `C++` with an uppercase `C` and a full-stop.

Reply:
This isn't a doc comment, so I wouldn't enforce any style here. But I'll fix it anyway :)